spec.bs

```diff
@@ -374,6 +374,31 @@ To <dfn>unmask tokens</dfn> given [=key commitments=] |issuerKeys|, byte string
 1. Let |result| ([=list=] of byte strings) be the result of running |unmaskOperation| on |pretokens| and |response|.
 1. Return |result|.
 
+To <dfn>set private token property for request</dfn> given a {{RequestInfo}} |input|, a {{RequestInit}} |init| and a {{Request}} |request| run the following steps:
+1. If |init|["{{RequestInit/privateToken}}"] [=map/exists=]:
```

Comment:
nit: I wonder if you should move this check to the monkeypatch so that you only call this when there is a privateToken field to check?

Reply:
Yes, I think this improves a lot. Prevents the large if block the algorithm had. Fixed.